src/main/java/world/anhgelus/molehunt/Molehunt.java

```diff
@@ -23,34 +25,33 @@ public class Molehunt implements ModInitializer {
 
     public static final String MOD_ID = "molehunt";
     public static final Logger LOGGER = LoggerFactory.getLogger(MOD_ID);
+    public static final Config CONFIG = new Config(MOD_ID);
+
     public Game game;
 
     public static HashMap<ServerPlayerEntity, Boolean> timerVisibility = new HashMap<>();
 
     @Override
     public void onInitialize() {
         LOGGER.info("Initializing Molehunt");
+        LOGGER.info(String.valueOf(CONFIG.GAME_DURATION));
```

Comment:
Add more information about this log or remove this

Reply:
this was just for testing